src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_variantContentPricing.html.twig

```diff
@@ -1,31 +1,35 @@
+{% deprecated 'The "_variantContent.html.twig" template is deprecated since Sylius 1.7 and will be romove in Sylius 2.0, use "_pricing.html.twig instead"' %}
+
 {% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
 
-<div class="ui segment">
-    <div class="ui small header">
-        {{ 'sylius.ui.pricing'|trans }}
+{% if channelPricings is defined %}
+    {% include '@SyliusAdmin/Product/Show/_pricing.html.twig' %}
```

Comment:
So that none of the other variables leak there.

```suggestion
    {% include '@SyliusAdmin/Product/Show/_pricing.html.twig' with {'channelPricings': channelPricings} only %}
```